src/splitInteger.test.js

```diff
@@ -4,18 +4,31 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
+  const goals = splitInteger(17, 4);
+  const equalParts = goals.slice(0, 3);
 
+  expect(equalParts).toEqual([4, 4, 4]);
 });
 
 test(`should return a part equals to a value
   when splitting into 1 part`, () => {
+  const goals = splitInteger(9, 1);
 
+  expect(goals)
+    .toEqual([9]);
 });
 
 test('should sort parts ascending if they are not equal', () => {
+  const goals = splitInteger(17, 4);
+  const sort = goals.sort((a, b) => a - b);
 
+  expect(goals)
```

Comment:
Using `.toBe` for array comparison might not work as expected, because it checks for reference equality. Use `.toEqual` to check for value equality. Replace with `expect(goals).toEqual(sort);`.